data/news/platform/platform-2023-10.md

```diff
@@ -0,0 +1,115 @@
+  - Respect [flags] field in opam packages -- [ocaml/dune#9047](https://github.com/ocaml/dune/pull/9047)
+  - Improve invalid substitute error -- [ocaml/dune#8922](https://github.com/ocaml/dune/pull/8922)
+  - Correctly verify tarball checksums -- [ocaml/dune#8876](https://github.com/ocaml/dune/pull/8876)
+  - Improve locations of conversion errors -- [ocaml/dune#8828](https://github.com/ocaml/dune/pull/8828)
+  - Remove post deps -- [ocaml/dune#8834](https://github.com/ocaml/dune/pull/8834)
+  - Move solver env printing to own command -- [ocaml/dune#8819](https://github.com/ocaml/dune/pull/8819)
+  - Record installed directories in `dune-package` -- [ocaml/dune#8953](https://github.com/ocaml/dune/pull/8953)
+
+## **[opam]** Native Support for Windows in opam 2.2 ([W5](https://ocaml.org/docs/platform-roadmap#w5-manage-dependencies))
+
+**Contributed by:** @rjbou (OCamlPro), @kit-ty-kate (Tarides), @dra27 (Tarides), @AltGr (OCamlPro)
+
+**Why:** opam and `opam-repository` currently don't support Windows natively. This effectively makes OCaml a very niche candidate on Windows, as users either have to (1) not use a package manager or (2) use a fork of opam and the `opam-repository`. Making opam and the `opam-repository` compatible with Windows will make OCaml a better choice for Windows users and help us grow the community. (More people = more packages, more contributors, more testing, yada yada)
+
+**What:** Releasing opam 2.2 with native support for Windows and making the official `opam-repository` usable on Windows.
+
+**Activities:**
+- Not much to report on this month. More to come for the November update (finalisation of `setenv` and release of opam 2.2 alpha3).
+
+## **[`odoc`]** Add Search Capabilities to `odoc` ([W25](https://ocaml.org/docs/platform-roadmap#w25-generate-documentation))
+
+**Contributed by:** @panglesd (Tarides), @EmileTrotignon (Tarides), @julow (Tarides), @jonludlam (Tarides)
+
+**Why:** The in-package search added in OCaml.org's central package documentation has been very well received by the community and improves how users navigate and discover OCaml documentation. We're upstreaming it to `odoc` to bring it into the local documentation as well and provide more advanced features, like searching by type.
+
+**What:** We're adding support in `odoc` for pluging in a search engine! `odoc` provides the UI (a search bar) and will generate a search index (that can also be used to be integrated into other search engine like Elasticsearch). We're also building a default client-side search engine based on Sherlodoc.
+
+**Activities:**
+- We've merged the PR adding [new search capabilities](https://github.com/ocaml/odoc/pull/972)! :tada: This will ship in the upcoming version of `odoc` 2.4.0.
+- We are currently experimenting with building an `odoc` search engine based on [Sherlocode](https://github.com/art-w/sherlodoc) --[art-w/sherlodoc#4](https://github.com/art-w/sherlodoc/pull/4).
```

Comment:
Should this be "Sherlocode" or "Sherlodoc"? Although it says Sherlocode, it links to Sherlodoc. 
Just checking if that was intentional.

Reply:
```suggestion
- We are currently experimenting with building an `odoc` search engine based on [Sherlodoc](https://github.com/art-w/sherlodoc) --[art-w/sherlodoc#4](https://github.com/art-w/sherlodoc/pull/4).
```